### DISCUSSION TITLE
Moving libsnet to github

### DISCUSSION THREAD

**Discussion by magnusviri** [Radmind/radmind#345] [Announcements]
@umeditor and @moretension: you guys are still admins at [sourceforge.org](https://sourceforge.net/projects/libsnet/). I reached out to Wes awhile ago about something else and didn't get a response. At least one person is still using Radmind. I'm not using it but I don't mind keeping it working. To make it easier to keep it working, I would like to move libsnet to GitHub. I don't know who else to talk to about this. https://radmind.org/ still exists and we have no idea who is paying for it. I'm hoping you guys can give us an idea of how we could keep this project working, how to get control of radmind.org, or any other information that can help us out.

**Comment by umeditor** [comment]
Unfortunately, I don’t have anything to do with Radmind anymore. Feel free to do what you’d like with the code.
